app/bundles/FormBundle/Translations/en_US/validators.ini

```diff
@@ -10,6 +10,7 @@ mautic.form.submission.captcha.invalid="The answer to %label% is incorrect. Plea
 mautic.form.submission.email.invalid="The email is invalid."
 mautic.form.submission.url.invalid="The URL is invalid."
 mautic.form.submission.phone.invalid="Please use the following international phone number format [+][country code][subscriber number] for this field (eg: ‪+14028650000)."
+mautic.form.submission.email.donotsubmit.invalid="Cannot be sent with this email"
```

Comment:
Is that variable (in form email field)? That should be a customizable text on the form if the feature is enabled on the form itself. The feature should be togglable form by form (for instance, no issue for newsletter subscription, but not acceptable to get pricelist).

Reply:
It's default, customizable text